player/dailymotion.coffee

```diff
@@ -18,7 +18,12 @@ window.DailymotionPlayer = class DailymotionPlayer extends Player
             if quality != 'auto'
                 params.quality = quality
 
-            @dm = DM.player('ytapiplayer',
+            @element = DM.$('ytapiplayer')
+            if not @element or @element.nodeType != Node.ELEMENT_NODE
+                throw new Error("Invalid player element in DailymotionPlayer(), requires an existing HTML element: " + @element)
```

Comment:
Under what conditions could this happen?  If this could realistically happen to a user it'd be better to add a warning banner on the page rather than throw an error from the script in which case the user probably won't realize what happened.